scan_filter_chain:
- name: shadows
  type: laser_filters/ScanShadowsFilter
  params:
        min_angle: 10
        max_angle: 170
        neighbors: 10
        window: 1
- name: range
  type: laser_filters/LaserScanRangeFilter
  params:
        lower_threshold: 0.3
        upper_threshold: 15
